pootle/templates/admin/detailed_reports.html

```diff
@@ -167,11 +167,11 @@
               {% endif %}
             </td>
             <td class="number">{{ score.wordcount }}</td>
-            <td class="number">{% if score.is_similarity_taken_from_mt %}<sup>MT</sup>{% endif %}{% if score.action == 'Review' %}&mdash;{% elif score.action == 'Translation' %}{% if score.similarity == 0 %}&lt; 50{% else %}{{ score.similarity|floatformat }}{% endif %}%{% endif %}</td>
+            <td class="number">{% if score.is_similarity_taken_from_mt %}<sup>MT</sup>{% endif %}{% if score.action == 'Review' %}&mdash;{% elif score.action == 'Translation' or score.action == 'Suggestion' %}{% if score.similarity == 0 %}&lt; 50{% else %}{{ score.similarity|floatformat }}{% endif %}%{% endif %}</td>
             <td class="number">{{ score.words|floatformat:2 }}</td>
             {% if object.rate > 0 and own_report or object.rate > 0 and admin_report %}
-              <td class="number">{% if score.action == 'Review' %}{{ score.review_rate }}{% elif score.action = 'Translation' %}{{ score.rate }}{% endif %} {{ object.currency }}</td>
-              <td class="number">{{ score.subtotal|floatformat:4 }} {{ object.currency }}</td>
+              <td class="number">{% if score.action != 'Suggestion' %}{% if score.action == 'Review' %}{{ score.review_rate }}{% elif score.action = 'Translation' %}{{ score.rate }}{% endif %} {{ object.currency }} {% else %}&mdash;{% endif %}</td>
+              <td class="number">{% if score.action != 'Suggestion' %}{{ score.subtotal|floatformat:4 }} {{ object.currency }}{% else %}&mdash;{% endif %}</td>
```

Comment:
The value for `score.action` is marked for l10n, therefore this code will never work in non-English UIs.

Also realize that it's a terrible idea to make comparisons against UI strings — rather use any sort of ID for that. Realize that this also affects more code in this template which is not being displayed in the diff.